base/src/org/compiere/acct/Doc.java

```diff
@@ -766,7 +766,7 @@ private final String postLogic (int index)
 			return STATUS_PeriodClosed;
 
 		if (isReversed() && IsReverseGenerated() && isReverseWithOriginalAccounting())
-			return generateReverseWithOriginalAccounting();
+			return generateReverseWithOriginalAccounting(m_ass[index]);

```

Comment:
Please will be good if you can refectory the crypt variables names.

m_ass[index] should be change by accountingSchemes[index]